website/docs/src/content/cloud/projects-environments.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Projects and environments
+description: Organize your GenSX workflows and deployments
+---
+
+# Projects and environments
+
+GenSX organizes your workflows and deployments using a flexible structure of projects and environments, making it easy to match the rest of your application architecture and CI/CD topology.
+
+## Project structure
+
+A project in GenSX is a collection of related workflows that are deployed, managed, and monitored together:
+
+- **Projects as logical units**: Group related workflows that serve a common purpose
+- **Shared configuration**: Apply settings across all workflows in a project
+- **Collective deployment**: Deploy all workflows within a project in one operation
+- **Unified monitoring**: View traces and metrics for an entire project
+
+Projects typically correspond to a codebase or application that contains multiple workflows.
+
+## Environment separation
+
+Within each project, you can have multiple environments:
+
+- **Development**: For building and testing new features
+- **Staging**: For pre-production validation
+- **Production**: For live, user-facing workflows
+- **Custom environments**: Create any specialized environments you need
```

Comment:
Same feedback as on another comment. This makes me think that gensx automatically creates these three envs for me. 

```suggestion
Within each project, you can have multiple environments. For example, you could create three environments for each project:

- **Development**: For building and testing new features
- **Staging**: For pre-production validation
- **Production**: For live, user-facing workflows

You have full control over your environments so you can organize them however you see fit. 
```